src/cases/repositories/case.repository.test.js

```diff
@@ -1,8 +1,22 @@
 import { describe, expect, it, vi } from "vitest";
 import { wreck } from "../../common/wreck.js";
-import { findAll, findById, updateStage } from "./case.repository.js";
-
-vi.mock("../../common/wreck.js");
+import {
+  completeStage,
+  findAll,
+  findById,
+  updateTaskStatus,
+} from "./case.repository.js";
+
+// Mock the wreck dependency
+const mockWreck = vi.hoisted(() => ({
+  get: vi.fn(),
+  post: vi.fn(),
+  patch: vi.fn(),
+}));
+
+vi.mock("../../common/wreck.js", () => ({
+  wreck: mockWreck,
+}));
```

Comment:
No need to manually stub. Let's rely on vitest to tell if imports are not defined